pino.d.ts

```diff
@@ -249,6 +249,7 @@ declare namespace pino {
     interface StreamEntry {
         stream: DestinationStream
         level?: Level
+        maxLevel?: Level
```

Comment:
Can you add a test for `maxLevel` property.

Reply:
Done